_includes/navbar.html

```diff
@@ -0,0 +1,31 @@
+<nav class="navbar navbar-toggleable-md navbar-inverse bg-faded">
+  <button class="navbar-toggler navbar-toggler-right" type="button" data-toggle="collapse" data-target="#myNav" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
```

Comment:
Is this supposed to toggle the navbar on smaller screens? (It shows up on my computer, even though it's wide enough, and clicking it just kind of glitches the visibility of the navbar options)

Also maybe this will conflict with #7